### PR TITLE
Devops2022 blogpost

### PR DESCRIPTION
**Pull request by martyjhenderson**
This PR:
* Adds in two image resources references
* Brings in a blog post about the State of DevOps 2022 report, focusing on the J-Curve

### CODE REVIEW

File: _posts/2023-01-03-state-of-devops.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Acknowledge the J-Curve
+
+One thing we often read in headline and in blog posts are about how lean little startups can produce endless work and enjoy mountains of unchained success. How many of those startups keep that momentum though? Ever startups that have gone big and stayed around, like Uber and AirBnB, have had slumps before resuming growth. However, most companies that are the pillars of success are not startups, but instead established companies with decades of experience behind them. Our clients range from start ups to these titans but they all share one common thing noted in this report - the J-Curve.
```
Comment (timotab) [inline _posts/2023-01-03-state-of-devops.md:18]: Typo: "Even startups"

**Comment by nerddtvg** [comment]
The blog post isn't published in the preview because the date is in the future. Can you update the date to today or yesterday so that we can see it rendered?

File: _posts/2023-01-03-state-of-devops.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+However, I bought a camper a few month later. If you have never tried to pull 20' of camper behind a vehicle (without swaybars), it's a new experience. Instead of the normal 2ish hours home, it took nearly 3 of going slower, stopping and checking to make sure things were attached, and finding a gas station I could pull through with the camper on it. My normal efficiency of driving plummeted by 50% while bringing the camper back. 
+
+The ability to camp without a tent was my goal, however, and so with a lot more experience of driving the truck and the camper places I got faster and better at it. At this point, I am pretty confident of driving the camper wherever and enjoy camping again. Thus, my goal of enjoying and camping more often and being efficient have been met, but there was a solid 18 months of learning to drive the truck differently (especially in the snow up here in Wisconsin), pulling a trailer, and generally adapting to an entire new way to go place.
```
Comment (timotab) [inline _posts/2023-01-03-state-of-devops.md:34]: typo: "to go places."

File: _posts/2023-01-03-state-of-devops.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+This is how the J-Curve works, not only did I need to learn a new tool (driving the truck) and a new technique (moving the camper), I also had to get over the fear of sliding the camper into other cars, thinking ahead on my turns, and planning places to take the camper. However, today in 2023, I can take the camper many places and my real concern is did I remember to pack everything and can drive my truck pretty efficiently. As you look to adapt to new tools to ensure reliability and the ability to pull your organization to your vision, just realize there will be a time they slow down and learn and that the whole organization will need to adapt to a new way of thinking. You can't just jump from a little car to a truck-and-trailer nor can you jump from a classic software desire to a powerful, robust, and scalable one without several months of adapting to it.
```
Comment (timotab) [inline _posts/2023-01-03-state-of-devops.md:36]: This sentence is clunky: "However, today in 2023, I can take the camper many places and my real concern is did I remember to pack everything and can drive my truck pretty efficiently."  Not sure what you're trying to say here. 

Typo: "classic software design", maybe?

**Comment by martyjhenderson** [comment]
> The blog post isn't published in the preview because the date is in the future. Can you update the date to today or yesterday so that we can see it rendered?

Fixed to today, but none of @timotab's comments applied yet

**Review by timotab — APPROVED** [review]

**Comment by martyjhenderson** [comment]
Now with Timothy's proofreading applied.

**Review by nerddtvg — APPROVED** [review]

**Pull request merged**